istioctl: fix `istioctl operator remove` fail when not specify reversion

my-git9 opened the pull request:
**Please provide a description of this PR:**
Fix: https://github.com/istio/istio/issues/42159

my-git9 committed:
istioctl: fix istioctl operator remove fail when a revision was not specified

Signed-off-by: xin.li <xin.li@daocloud.io>

istio-testing commented:
@my-git9: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 836beb4f46f571631bf2eb0e0539d10fba873c2d | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/release-notes_istio/1596085639781027840) | true | `/test release-notes`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

my-git9 commented:
The test result:
```shell
./istioctl operator init  --tag 1.15.1 --hub docker.io/istio -r default
Operator controller is already installed in istio-operator namespace.
Upgrading operator controller in namespace: istio-operator using image: docker.io/istio/operator:1.15.1
Operator controller will watch namespaces: istio-system
✔ Istio operator installed
✔ Installation complete
$
$
$ kubectl -nistio-system get po
NAME                             READY   STATUS             RESTARTS      AGE
istio-operator-7fb9cc79d-8h4qm   0/1     CrashLoopBackOff   3 (35s ago)   79s
$
$ 
$ ./istioctl operator remove
Istio operator revision default will be removed from cluster, Proceed? (y/N) y
Removing Istio operator...
  Removed Deployment:istio-operator:istio-operator.
  Removed Service:istio-operator:istio-operator.
  Removed ServiceAccount:istio-operator:istio-operator.
  Removed ClusterRole::istio-operator.
  Removed ClusterRoleBinding::istio-operator.
✔ Removal complete
$ kubectl get po -n istio-operator
No resources found in istio-operator namespace.
```

zirain commented:
can you add a test/e2e for this?

istio-testing commented:
@my-git9: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
unit-tests-arm64_istio | 99a9886c632d113d2d38f3666ad8eae8ae9b7fc3 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/unit-tests-arm64_istio/1597207543287910400) | true | `/test unit-tests-arm64`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
unit-tests-arm64_istio | 99a9886c632d113d2d38f3666ad8eae8ae9b7fc3 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/unit-tests-arm64_istio/1597207543287910400) | true | `/test unit-tests-arm64`
unit-tests_istio | 99a9886c632d113d2d38f3666ad8eae8ae9b7fc3 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/unit-tests_istio/1597207543241773056) | true | `/test unit-tests`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
unit-tests-arm64_istio | 99a9886c632d113d2d38f3666ad8eae8ae9b7fc3 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/unit-tests-arm64_istio/1597207543287910400) | true | `/test unit-tests-arm64`
unit-tests_istio | 99a9886c632d113d2d38f3666ad8eae8ae9b7fc3 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/unit-tests_istio/1597207543241773056) | true | `/test unit-tests`
gencheck_istio | 99a9886c632d113d2d38f3666ad8eae8ae9b7fc3 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/gencheck_istio/1597207554113409024) | true | `/test gencheck`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

my-git9 commented:
/retest

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
unit-tests_istio | 99a9886c632d113d2d38f3666ad8eae8ae9b7fc3 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/unit-tests_istio/1597207543241773056) | true | `/test unit-tests`
unit-tests-arm64_istio | 99a9886c632d113d2d38f3666ad8eae8ae9b7fc3 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/unit-tests-arm64_istio/1597210643662901248) | true | `/test unit-tests-arm64`
gencheck_istio | 99a9886c632d113d2d38f3666ad8eae8ae9b7fc3 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/gencheck_istio/1597210643700649984) | true | `/test gencheck`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

my-git9 commented:
/retest

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
gencheck_istio | 99a9886c632d113d2d38f3666ad8eae8ae9b7fc3 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/gencheck_istio/1597210643700649984) | true | `/test gencheck`
unit-tests_istio | 99a9886c632d113d2d38f3666ad8eae8ae9b7fc3 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/unit-tests_istio/1597210643620958208) | true | `/test unit-tests`
unit-tests-arm64_istio | 99a9886c632d113d2d38f3666ad8eae8ae9b7fc3 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/unit-tests-arm64_istio/1597212598682521600) | true | `/test unit-tests-arm64`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

my-git9 commented:
> can you add a test/e2e for this?

Sorry, I didn't find e2e test cases for installing, uninstalling (including `istioctl install` `istioctl uninstall`  `istioctl operator init` `istioctl operator remove`) from the istio repository.

zirain commented:
can you try something like https://github.com/istio/istio/blob/master/tests/integration/operator/uninstall_test.go

my-git9 commented:
> can you try something like https://github.com/istio/istio/blob/master/tests/integration/operator/uninstall_test.go

Thansks, I wiil try it later

istio-testing commented:
@my-git9: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/lint_istio/1599741335823192064) | true | `/test lint`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/lint_istio/1599741335823192064) | true | `/test lint`
gencheck_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/gencheck_istio/1599741336653664256) | true | `/test gencheck`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/lint_istio/1599741335823192064) | true | `/test lint`
gencheck_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/gencheck_istio/1599741336653664256) | true | `/test gencheck`
integ-operator-controller_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/integ-operator-controller_istio/1599741323244474368) | true | `/test integ-operator-controller`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/lint_istio/1599741335823192064) | true | `/test lint`
gencheck_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/gencheck_istio/1599741336653664256) | true | `/test gencheck`
integ-operator-controller_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/integ-operator-controller_istio/1599741323244474368) | true | `/test integ-operator-controller`
analyze-tests_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/analyze-tests_istio/1599741334984331264) | true | `/test analyze-tests`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/lint_istio/1599741335823192064) | true | `/test lint`
gencheck_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/gencheck_istio/1599741336653664256) | true | `/test gencheck`
analyze-tests_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/analyze-tests_istio/1599741334984331264) | true | `/test analyze-tests`
integ-operator-controller_istio | 55dcd4a3256fc8ccd70be162e91c6917b0dd4dac | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/integ-operator-controller_istio/1599745253475094528) | true | `/test integ-operator-controller`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
analyze-tests_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/analyze-tests_istio/1599741334984331264) | true | `/test analyze-tests`
lint_istio | 55dcd4a3256fc8ccd70be162e91c6917b0dd4dac | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/lint_istio/1599745265214951424) | true | `/test lint`
integ-operator-controller_istio | b160897739e1b1247f0d787183a6308a88c1244e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/integ-operator-controller_istio/1599747017054097408) | true | `/test integ-operator-controller`
gencheck_istio | b160897739e1b1247f0d787183a6308a88c1244e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/gencheck_istio/1599747029632815104) | true | `/test gencheck`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
analyze-tests_istio | 8100dbf0d703dbab067348eb6d1139fea2f52994 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/analyze-tests_istio/1599741334984331264) | true | `/test analyze-tests`
integ-operator-controller_istio | b160897739e1b1247f0d787183a6308a88c1244e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/integ-operator-controller_istio/1599747017054097408) | true | `/test integ-operator-controller`
gencheck_istio | b160897739e1b1247f0d787183a6308a88c1244e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/gencheck_istio/1599747029632815104) | true | `/test gencheck`
lint_istio | b160897739e1b1247f0d787183a6308a88c1244e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/lint_istio/1599747028793954304) | true | `/test lint`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
integ-operator-controller_istio | b160897739e1b1247f0d787183a6308a88c1244e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/integ-operator-controller_istio/1599747017054097408) | true | `/test integ-operator-controller`
gencheck_istio | b160897739e1b1247f0d787183a6308a88c1244e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/gencheck_istio/1599747029632815104) | true | `/test gencheck`
lint_istio | b160897739e1b1247f0d787183a6308a88c1244e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/lint_istio/1599747028793954304) | true | `/test lint`
analyze-tests_istio | b160897739e1b1247f0d787183a6308a88c1244e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/analyze-tests_istio/1599747027959287808) | true | `/test analyze-tests`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
integ-operator-controller_istio | c4649824cf96c970672b6d6f315d56b3c2e423cf | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/integ-operator-controller_istio/1599752370244292608) | true | `/test integ-operator-controller`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 917164ba491783eda5bf1943f542abd7a29bc519 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/lint_istio/1599754346495479808) | true | `/test lint`
integ-operator-controller_istio | fd5dfe24d62f83ce395e82c646e95d793450f06a | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/integ-operator-controller_istio/1599755627591110656) | true | `/test integ-operator-controller`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
integ-operator-controller_istio | fd5dfe24d62f83ce395e82c646e95d793450f06a | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/integ-operator-controller_istio/1599755627591110656) | true | `/test integ-operator-controller`
analyze-tests_istio | c3cb7d47247575b8874a42b11262bff124f60b82 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/analyze-tests_istio/1599756724015730688) | true | `/test analyze-tests`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
analyze-tests_istio | c3cb7d47247575b8874a42b11262bff124f60b82 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/analyze-tests_istio/1599756724015730688) | true | `/test analyze-tests`
integ-operator-controller_istio | c3cb7d47247575b8874a42b11262bff124f60b82 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/integ-operator-controller_istio/1599756713148289024) | true | `/test integ-operator-controller`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@my-git9: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
integ-operator-controller_istio | be2dc57dd6bd3bcb520f093d2a42ae8c3415f2e4 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42160/integ-operator-controller_istio/1599760880331919360) | true | `/test integ-operator-controller`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

my-git9 commented:
PTAL @istio/wg-test-and-release-maintainers @istio/release-managers @istio/wg-environments-maintainers

my-git9 commented:
PTAL @zirain @ericvn @howardjohn @litong01

zirain reviewed (approved):
lgtm, thanks

ericvn commented on releasenotes/notes/42160.yaml:
```diff
@@ -0,0 +1,8 @@
+  - 42160
+releaseNotes:
+  - |
+    **Fixed** `istioctl operator remove` failed when not specify revision.
```
```suggestion
    **Fixed** `istioctl operator remove` failed when a revision was not specified.
```

my-git9 replied:
Updated, thanks
